checker/tests/nullness/Issue1142.java

```diff
@@ -0,0 +1,14 @@
+// Issue 1142 https://github.com/typetools/checker-framework/issues/67
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Issue1142 {
+
+    void foo() {
+
+        ConcurrentHashMap<Integer, @Nullable Integer> chm = new ConcurrentHashMap<>();
```

Comment:
The point of Issue 1142 was that you don't want to allow nullable keys or values. So this line should give an error.